packages/smooth_app/lib/pages/offline_tasks_page.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:flutter/material.dart';
+import 'package:smooth_app/generic_lib/duration_constants.dart';
+import 'package:task_manager/task_manager.dart';
+
+// TODO(ashaman999): add the translations later
+class OfflineTask extends StatefulWidget {
+  const OfflineTask({
+    super.key,
+  });
+
+  @override
+  State<OfflineTask> createState() => _OfflineTaskState();
+}
+
+class _OfflineTaskState extends State<OfflineTask> {
+  Future<List<Task>> _fetchListItems() async {
+    final Iterable<Task> tasks = await TaskManager().listPendingTasks();
+    return tasks.toList(growable: true);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Pending Background Tasks'),
+        actions: <Widget>[
+          PopupMenuButton<int>(
+            onSelected: (int item) async {
+              await _cancelAllTask();
+            },
+            itemBuilder: (BuildContext context) => <PopupMenuItem<int>>[
+              const PopupMenuItem<int>(value: 0, child: Text('Cancel all')),
+            ],
+          ),
+        ],
+        leading: BackButton(
+          onPressed: () => Navigator.pop(context),
+        ),
+      ),
+      body: RefreshIndicator(
+        onRefresh: () async {
```

Comment:
Async is useless here

Reply:
It's showing a warning here, can't remove as of now

Reply:
You're a Future is expected here.